src/fiatExchanges/CoinbasePaymentSection.tsx

```diff
@@ -13,28 +13,43 @@ import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
+import { useTokenInfo } from 'src/tokens/hooks'
 import { walletAddressSelector } from 'src/web3/selectors'
+import { getNetworkFromNetworkId } from 'src/web3/utils'
 
 export interface CoinbasePaymentSectionProps {
   cryptoAmount: number
   coinbaseProvider: FetchProvidersOutput
   appId: string
   analyticsData: ProviderSelectionAnalyticsData
+  tokenId: string
 }
 
 export function CoinbasePaymentSection({
   cryptoAmount,
   coinbaseProvider,
   appId,
   analyticsData,
+  tokenId,
 }: CoinbasePaymentSectionProps) {
   const { t } = useTranslation()
   const walletAddress = useSelector(walletAddressSelector)!
+  const tokenInfo = useTokenInfo(tokenId)
 
+  if (!tokenInfo) {
+    // should never happen
```

Comment:
A logger error here would be nice so that we known that something that should never happen has happened.